app/src/main/java/io/capibaras/abcall/data/repositories/IncidentsRepository.kt

```
@@ -90,9 +93,37 @@ class IncidentsRepository(
+                    escalatedDate = getEscalatedDate(incident.history),
+                    closedDate = getClosedDate(incident.history),
+                    recentlyUpdated = setRecentlyUpdated(incident)
+                )
+
+                incidentDAO.insertIncident(incident)
+
+                Result.success(incident)
+            } else {
+                handleErrorResponse(localData, response.errorBody()?.string())
+            }
+        } catch (e: IOException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)